lib/Segments/DynamicSegments/Filters/SubscriberSubscribedDate.php

```diff
@@ -21,13 +21,13 @@ public function apply(QueryBuilder $queryBuilder, DynamicSegmentFilterEntity $fi
     $operator = $filterData->getParam('operator');
 
     if ($operator === self::BEFORE) {
-      $queryBuilder->andWhere('last_subscribed_at < :date'. $filter->getId());
+      $queryBuilder->andWhere('last_subscribed_at < :date' . $filter->getId());
```

Comment:
Maybe using a variable for the parameter name would make the code more readable.

```
$dateParameter = 'date' . $filter->getId();
$queryBuilder->andWhere("last_subscribed_at < :$dateParameter");
```